posts/2018-01-07-how-to-read.md

```diff
@@ -0,0 +1,46 @@
+---
+title: How to Read
+---
+
+I've been told the rate and retention I have while reading is abnormal to say
+the least.  Others have been interested in trying my reading methodology and
+seeing if it would help them read better.  As a result, I've been making an
+attempt to codify how I read so that others can try it for themselves.
+
+The first idiosyncracy to my methodology is the concurrency I utilize while
+reading books.  Since college I've always read around five to ten books
+concurrently.  Generally, this is received with awe and intrigue about how I'm
+able to keep them separate or retain any number of them let alone one.
+
+What I've found over the years is that by reducing the amount of information
+consumed on a topic in a given sitting, it increases the ability to let that
+knowledge settle into long-term memory (references).  Thus, one of the ways to
+increase concurrency is to read a section or chapter of one book and then read a
+section or chapter of the next and repeat until done reading for the session.
+
+I couple short reading bursts of an individual book with practised recall before
+continuing a book I read in a previous session.  For example, in one session I
+might read "Type-Driven Development with Idris", "The Thirteen Books of the
+Elements", and "Abstract Algebra".  In my next reading session, I would take a
+moment before starting to recall the section or chapter I previously read of
+"Type-Driven Development with Idris".  Then I would read the next section or
+chapter before doing the same recall and reading of "The Thirteen Books of the
+Elements".
+
+By re-establishing reading context without re-reading the previous section
+before continuing a book, I ensure that I can keep the contents of a book
+relevant to some mnemonic.  Usually the author, title, or cover if not all
+three.
```

Comment:
The last sentence is a sentence fragment. Remove the period after mnemonic and lowercase usually.